src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
+    if [[ "${OS}" = "Darwin" ]]; then
+        local mac_version=$(sw_vers -productVersion)
+        if [[ $(awk 'BEGIN {print ("'${mac_version}'" < "'10.16'")}') )) ]]; then
+            die "WARNING: MacOS version >= 10.13 is required for all Confluent features to work."
```

Comment:
`MacOS` --> `macOS`

Reply:
Should `Confluent` be `Confluent Platform`?

Reply:
I like to sometimes see what went wrong in the same message. 
Suggestion:
`die "Error: MacOS version >= 10.13 is required for all Confluent features to work (detected MacOS: ${mac_version})"`

Also suggesting to call this an `Error` since execution does not proceed beyond this point.

Reply:
Thanks, done!